File: backend/src/test/java/com/skewwhiffy/auraltester/helper/StringHelperTest.java
package com.skewwhiffy.auraltester.helper;

import lombok.val;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class StringHelperTest {
    @Test
    void when_getTitleCase_then_eachWordIsCapitalized() {
        val source = "this is a title";
        val expected = "This Is A Title";

        val actual = StringHelper.getTitleCase(source.split(" "));

        assertThat(actual).isEqualTo(expected);
    }
}

File: backend/src/test/kotlin/com/skewwhiffy/auraltester/helper/StringHelperTest.kt
package com.skewwhiffy.auraltester.helper

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class StringHelperTest {
    @Test
    fun when_getTitleCase_then_eachWordIsCapitalized() {
        val source = "this is a title"
        val expected = "This Is A Title"

        val actual = getTitleCase(*source.split(" ").toTypedArray())

        assertThat(actual).isEqualTo(expected)
    }
}
